src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsServiceImpl.java

```diff
@@ -281,6 +278,16 @@ private boolean isoCountryCodeTranslatesToSingleFipsCountryCode(String isoCountr
         }
     }
     
+    private boolean bankIsoCountryCodeTranslatesToSingleFipsCountryCodeWhenBankDataIsEntered(String isoCountryCode, List<String> errorMessages) {
+        if (ObjectUtils.isNotNull(isoCountryCode)) {
```

Comment:
Should this be checking for a blank value, instead of just a null one? Or do you need the country-code-checking to proceed for empty or blank values?

Reply:
This should check for just null. The country code check needs to continue so that the error messages for the report are generated for the empty and blank values.